### ISSUE TITLE
Definition of Module

### ISSUE THREAD

**Issue by charleshalldesign** [w3c/coga#384]
In the [Cognitive Accessibility Research Modules](https://www.w3.org/TR/coga-research-modules/), the term module is used 33 times in the body of the document. While the purpose of a research module and what it might include are stated, the term itself is undefined. Importantly, per definitions available from various dictionaries, a module is a unit of information that can be used independently, or together as a collection. This document offers no insight into how these topics would or should be used together, such as the needs associated with Voice Systems and Indoor Wayfinding. This is important in the cognitive accessibility space, given that there exist both compound effects and cumulative effects of multiple needs occurring at once or in succession.

Define module and clarify use independently, and/or as a collection.

**Comment by majidhussaun2345-wq** [comment]
### - ###

**Comment by majidhussaun2345-wq** [comment]
### divorce in Washington state is the dissolution of marriage. Our firm’s founder, Veronica Freitas, is skilled at working with individuals who are contemplating or going through a divorce and are experiencing one of the most difficult periods in their lives.

At V. Freitas Law, we are sensitive to the needs of our divorce clients

**Comment by charleshalldesign** [comment]
> ### divorce in Washington state is the dissolution of marriage. Our firm’s founder, Veronica Freitas, is skilled at working with individuals who are contemplating or going through a divorce and are experiencing one of the most difficult periods in their lives.
> At V. Freitas Law, we are sensitive to the needs of our divorce clients

This seems to have been pasted from an unrelated source or topic.
